gears/src/rest/pagination.rs

```diff
@@ -1,21 +1,39 @@
-use serde::Deserialize;
+use crate::types::pagination::request::{PaginationRequest, QUERY_DEFAULT_LIMIT};
 
-const QUERY_DEFAULT_LIMIT: u8 = 100;
-
-//#[derive(FromForm, Debug)]
-#[derive(Deserialize, serde::Serialize, Debug, Clone, Eq, PartialEq)]
+#[derive(serde::Deserialize, serde::Serialize, Debug, Clone, Eq, PartialEq)]
 pub struct Pagination {
-    offset: Option<u32>,
+    /// offset is a numeric offset that can be used when key is unavailable.
+    /// It is less efficient than using key. Only one of offset or key should
+    /// be set.
+    pub offset: u32,
```

Comment:
Doesn't this change the API?

Reply:
How? Behavior same, except now you write `Option<Pagination>`

Reply:
I think a query like this would have worked previously: `?offset=10` i.e. the offset would be set to ten. Now I assume the offset would be set to default?

Reply:
Hm. I see what you mean. You right and for rest it would be better to have optional fields, or use `serde(default)`

Reply:
After a minute I think that for rest `Pagination` with `Option` fields would be okay.